include/pmacc/particles/ParticlesBase.hpp

```diff
@@ -172,6 +172,11 @@ class ParticlesBase : public SimulationFieldHelper<T_MappingDesc>
     /** copy gard particles to intermediate exchange buffer
      *
      * Copy all particles from the guard of a direction to the device exchange buffer.
+     *  @warning This method resets the number of particles in the processed supercells even
+     * if there are particles left in the supercell and not fulfill the rule that the
+     * last frame is contiguous filled.
+     * Call KernelFillGaps afterwards if you need a valid number of particles
```

Comment:
I'm not sure what style is used in picongpu for such things, but maybe `KernelFillGaps` should be written as `KernelFillGaps()` to emphasize it is a function?